packages/eslint-config-skuba/index.js

```diff
@@ -1,151 +1,188 @@
+      'coverage*/',
+      'dist*/',
+      'lib*/',
+      'tmp*/',
+    ],
+  },
+  ...base.map(({ plugins: { jest: _jest, ...restPlugins } = {}, ...conf }) => ({
+    ...conf,
+    plugins: restPlugins,
+  })),
+  {
+    rules: {
+      'import-x/no-duplicates': 'error',
+
+      'import-x/order': [
+        'error',
+        {
+          alphabetize: {
+            order: 'asc',
+          },
+
+          'newlines-between': 'always',
+
+          pathGroups: [
+            {
+              group: 'external',
+              pattern: 'src',
+              position: 'after',
+            },
+            {
+              group: 'external',
+              pattern: 'src/**',
+              position: 'after',
+            },
+          ],
+
+          pathGroupsExcludedImportTypes: ['builtin'],
+        },
+      ],
+
+      'jest/expect-expect': 'off',
+      'jest/no-deprecated-functions': 'error',
+      'jest/prefer-expect-resolves': 'error',
+      'jest/prefer-spy-on': 'error',
+      'jest/prefer-strict-equal': 'off',
+      'jest/prefer-to-be': 'error',
+      'jest/prefer-to-contain': 'error',
+      'jest/prefer-to-have-length': 'error',
+      'jest/prefer-todo': 'error',
+      'jest/valid-title': 'error',
+      'no-use-before-define': 'off',
+
+      quotes: [
+        'warn',
+        'single',
+        {
+          allowTemplateLiterals: false,
+          avoidEscape: true,
+        },
+      ],
+
+      'sort-imports': [
+        'error',
+        {
+          ignoreDeclarationSort: true,
+        },
       ],
-      files: [`*.{${ts.join(',')}}`],
+    },
+  },
+  ...compat
+    .extends(
+      'plugin:@typescript-eslint/recommended-type-checked',
+      'plugin:@typescript-eslint/stylistic-type-checked',
+    )
+    .map((config) => ({
+      ...config,
+      files: [`**/*.{${tsExtensions}}`],
+    })),
+  {
+    files: [`**/*.{${tsExtensions}}`],
+
+    languageOptions: {
+      ecmaVersion: 5,
+      sourceType: 'script',
+
       parserOptions: {
-        project: true,
+        projectService: true,
       },
-      rules: {
-        '@typescript-eslint/consistent-type-exports': 'error',
-        '@typescript-eslint/no-floating-promises': 'error',
-        // allow || on strings and booleans
-        '@typescript-eslint/prefer-nullish-coalescing': [
-          'error',
-          {
-            ignorePrimitives: { string: true, boolean: true },
+    },
+
+    rules: {
+      '@typescript-eslint/consistent-type-exports': 'error',
+      '@typescript-eslint/no-floating-promises': 'error',
+
+      '@typescript-eslint/prefer-nullish-coalescing': [
+        'error',
+        {
+          ignorePrimitives: {
+            string: true,
+            boolean: true,
           },
-        ],
-        // prefer type assertions over null assertions
-        '@typescript-eslint/no-non-null-assertion': 'error',
-        '@typescript-eslint/non-nullable-type-assertion-style': 'off',
+        },
+      ],
 
-        // too spicy 🌶️
```

Comment:
🥹

Reply:
I can restore some of these comments, they got lost in the migration script